index.html

```diff
@@ -202,7 +202,7 @@ <h3>Expected Behavior</h3>
 		
   <li>Be aware that displays of affection may complicate professional relationships. For some cultures, overtly friendly disposition towards another participant involving body contact (e.g.: hugging, touching on the arm or shoulder, or kissing) is uncommon and may be perceived as an invasion of personal space, or as unwelcome advances. Work to eliminate your own biases, prejudices and discriminatory practices. </li>
 
-<li>Think of others’ needs from their point of view. Use preferred names, titles (including pronouns) and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others, at least until a truly friendly atmosphere and relationships are established.</li>
+<li>Think of others’ needs from their point of view. Use preferred names, titles (including pronouns) and the appropriate tone of voice. Therefore, be formal and conservative in what you do and liberal in what you accept from others and acknowledge the contributions of your peers. At least until a truly friendly atmosphere and relationships are established.</li>
```

Comment:
I am afraid that the "acknowledgement of contributions from your peers" is misplaced, it doesn't belong in this particular sentence where we talk about cultural differences and behavioral norms.

Reply:
This short comment is to fill in missing functionality from the old document which Nigel pointed out. It is for setting the tone of the conversation and seems to be the best place for it, since it is on the border between process and good behaviour. Under the subject of "Think of others’ needs from their point of view."